Coalition.rte/Devices/Weapons/MachineGuns/MuzzleSmoke.lua

```diff
@@ -0,0 +1,22 @@
+function Create(self)
```

Comment:
This looks super similar to the same muzzle smoke one again. Really should just have 1 script that supports multiple guns or something.